app/notification.go

```diff
@@ -431,7 +431,13 @@ func (a *App) sendNotificationEmail(post *model.Post, user *model.User, channel
 		// fall back to sending a single email if we can't batch it for some reason
 	}
 
+	var useMilitaryTime bool
 	translateFunc := utils.GetUserTranslations(user.Locale)
+	if result := <-a.Srv.Store.Preference().Get(user.Id, model.PREFERENCE_CATEGORY_DISPLAY_SETTINGS, "use_military_time"); result.Err != nil {
+		useMilitaryTime = true
```

Comment:
I left it to true because the previous implementation supported militaryTime timestamp.

Whatever works best.  We can default to false.  It would not fail if useMilitaryTime = false